Frontend/android-native/bundling.sh

```
@@ -1,21 +1,21 @@
-sh userJuspayAssets.sh $1
+# mkdir -p "app/src/user/$1/assets/juspay"
+# rm -rf app/src/user/$1/assets/juspay/index_bundle.js
+# cp ./../ui-customer/dist/android/index_bundle.js app/src/user/$1/assets/juspay
+# rm -rf app/src/user/$1/assets/juspay/juspay_assets.json
+# cd ..
+# sh userJuspayAssets.sh $1
 
 echo " ---------- Driver prod:android :- --------------"
 
-cd ui-driver
+cd ../ui-driver
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)